[2021-04-21 20:15:08,580] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: docker_sample.sleep 2021-04-19T07:10:00+00:00 [queued]>
[2021-04-21 20:15:08,608] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: docker_sample.sleep 2021-04-19T07:10:00+00:00 [queued]>
[2021-04-21 20:15:08,609] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-21 20:15:08,610] {taskinstance.py:1043} INFO - Starting attempt 1 of 4
[2021-04-21 20:15:08,611] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-21 20:15:08,625] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): sleep> on 2021-04-19T07:10:00+00:00
[2021-04-21 20:15:08,636] {standard_task_runner.py:52} INFO - Started process 520 to run task
[2021-04-21 20:15:08,649] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'docker_sample', 'sleep', '2021-04-19T07:10:00+00:00', '--job-id', '225', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dockerOperator.py', '--cfg-path', '/tmp/tmpikrpvm40', '--error-file', '/tmp/tmpdj9kjzcm']
[2021-04-21 20:15:08,654] {standard_task_runner.py:77} INFO - Job 225: Subtask sleep
[2021-04-21 20:15:08,716] {logging_mixin.py:104} INFO - Running <TaskInstance: docker_sample.sleep 2021-04-19T07:10:00+00:00 [running]> on host 5af2b1cab26e
[2021-04-21 20:15:08,791] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=docker_sample
AIRFLOW_CTX_TASK_ID=sleep
AIRFLOW_CTX_EXECUTION_DATE=2021-04-19T07:10:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-19T07:10:00+00:00
[2021-04-21 20:15:08,792] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-04-21 20:15:08,794] {bash.py:158} INFO - Running command: sleep 5
[2021-04-21 20:15:08,806] {bash.py:169} INFO - Output:
[2021-04-21 20:15:13,809] {bash.py:177} INFO - Command exited with return code 0
[2021-04-21 20:15:13,920] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=docker_sample, task_id=sleep, execution_date=20210419T071000, start_date=20210421T201508, end_date=20210421T201513
[2021-04-21 20:15:13,996] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-04-21 20:15:14,039] {local_task_job.py:146} INFO - Task exited with return code 0
